pkg/virt-controller/watch/drain/evacuation/evacuation_test.go

```diff
@@ -445,6 +446,89 @@ var _ = Describe("Evacuation", func() {
 
 			controller.Execute()
 		})
+
+		It("Should not evict new VMIs when the amount of concurrent running migrations reached its maximum capacity", func() {
+			var maxParallelMigrationsPerCluster uint32 = 5
+			config, _, _ := testutils.NewFakeClusterConfigUsingKVConfig(&v1.KubeVirtConfiguration{
+				MigrationConfiguration: &v1.MigrationConfiguration{
+					ParallelMigrationsPerCluster: &maxParallelMigrationsPerCluster,
+				},
+			})
+
+			controller = evacuation.
+				NewEvacuationController(
+					vmiInformer,
+					migrationInformer,
+					nodeInformer,
+					podInformer,
+					recorder,
+					virtClient,
+					config)
+
+			nodeName := "node01"
+			addNode(newNode(nodeName))
+
+			vmi1 := newVirtualMachineMarkedForEviction("testvm1", nodeName)
+			vmiFeeder.Add(vmi1)
```

Comment:
Why do we need this vmi?

Reply:
@iholder-redhat You are correct, for the sake of consistency I will create a VMI and a corresponding VMIM. Will remove it.